addons/website_sale/views/templates.xml

```diff
@@ -874,7 +874,7 @@
         <t t-call="website.layout">
             <t t-set="no_footer" t-value="1"/>
             <div id="wrap">
-                <div class="container oe_website_sale py-2">
```

Comment:
are those changes about padding classes still needed?

Reply:
yes it is not inside modals

Reply:
About @qsm-odoo remarks https://github.com/odoo/odoo/pull/67732/files#r592982942
> It seems there is an oe_structure after that wrap so this fix would only work if nothing in that oe_structure

I think it is fine, as we can only (or mostly?) drag & drop sections inside oe_structure and those have already a padding.

Reply:
The point was the design would be strange with a huge padding on top of the oe_structure... but granted: it is probably already strange with a small padding